root/etc/supervisord.conf

```diff
@@ -0,0 +1,163 @@
+[unix_http_server]
+file=/tmp/supervisor.sock   ; (the path to the socket file)
+
+[supervisord]
+;logfile=/tmp/supervisord.log ; (main log file;default.conf $CWD/supervisord.log)
+logfile=/dev/null
+logfile_maxbytes=0        ; (max main logfile bytes b4 rotation;default.conf 50MB)
+logfile_backups=10           ; (num of main logfile rotation backups;default.conf 10)
+loglevel=info                ; (log level;default.conf info; others: debug,warn,trace)
+pidfile=/tmp/supervisord.pid ; (supervisord pidfile;default.conf supervisord.pid)
+nodaemon=false               ; (start in foreground if true;default.conf false)
+minfds=1024                  ; (min. avail startup file descriptors;default.conf 1024)
+minprocs=200                 ; (min. avail process descriptors;default.conf 200)
+user=root		             ;
+
+; the below section must remain in the config file for RPC
+; (supervisorctl/web interface) to work, additional interfaces may be
+; added by defining them in separate rpcinterface: sections
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl=unix:///tmp/supervisor.sock ; use a unix:// URL  for a unix socket
+
+[program:appwrite]
+command=php /usr/src/code/app/http.php -dopcache.preload=opcache.preload=/usr/src/code/app/preload.php
```

Comment:
I would add a list of all the logs files and how to view them if needed.
`docker-compose exec appwrite-lite cat /path/to/log`.